extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -326,152 +361,155 @@ public function doWikiForm() {
 			"protected"   => WikiFactory::getFlags ( $this->mWiki->city_id ) & WikiFactory::FLAG_PROTECTED,
 			"statuses"    => $this->mStatuses,
 			"variables"   => WikiFactory::getVariables(),
-			"variableName"=> $this->mVariableName,
+			"variableName" => $this->mVariableName,
 			"isDevel"     => $wgDevelEnvironment,
-			'wikiFactoryUrl' => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullUrl(),
+			'wikiFactoryUrl' => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullURL(),
 			'wgStylePath' => $wgStylePath,
-		);
-		if( $this->mTab === 'info' ) {
+		];
+		if ( $this->mTab === 'info' ) {
 			$vars[ 'founder_id' ] = $this->mWiki->city_founding_user;
-			#this is the static stored email
+			# this is the static stored email
 			$vars[ 'founder_email' ] = $this->mWiki->city_founding_email;
 
-			if( !empty( $this->mWiki->city_founding_user ) ) {
-				#if we knew who they were, get their current info
+			if ( !empty( $this->mWiki->city_founding_user ) ) {
+				# if we knew who they were, get their current info
 				$fu = User::newFromId( $this->mWiki->city_founding_user );
 				$vars[ 'founder_username' ] = $fu->getName();
 				$vars[ 'founder_usermail' ] = $fu->getEmail();
 				$vars[ 'founder_metrics_url' ] = $vars[ 'wikiFactoryUrl' ] . "/Metrics?founder=" . rawurlencode( $fu->getName() );
 				$vars[ 'founder_usermail_metrics_url' ] = $vars[ 'wikiFactoryUrl' ] . "/Metrics?email=" . urlencode( $vars[ 'founder_usermail' ] );
 				$vars[ 'founder_email_metrics_url' ] = $vars[ 'wikiFactoryUrl' ] . "/Metrics?email=" . urlencode( $vars[ 'founder_email' ] );
-			} else
-			{	#dont know who made the wiki, so dont try to do lookups
+			} else {
+				# We don't know who made the wiki, so don't try to do look ups
 				$vars[ 'founder_username' ] = null;
 				$vars[ 'founder_usermail' ] = null;
 			}
 
-			if( $wgUser->isAllowed( 'lookupuser' ) ) {
-				$vars[ 'lookupuser_by_founder_email_url' ] = Title::newFromText( "LookupUser", NS_SPECIAL)->getFullURL(array("target" => $vars['founder_email']));
+			if ( $wgUser->isAllowed( 'lookupuser' ) ) {
+				$vars[ 'lookupuser_by_founder_email_url' ] = Title::newFromText( "LookupUser", NS_SPECIAL )->getFullURL( [  "target" => $vars['founder_email']  ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "LookupUser" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "target" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)